timesketch/lib/analyzers/interface.py

```diff
@@ -1150,3 +1165,196 @@ def get_kwargs(cls):
     def run(self):
         """Entry point for the analyzer."""
         raise NotImplementedError
+
+
+class AnalyzerOutputException(Exception):
+    """Analyzer output exception."""
+
+
+class AnalyzerOutput:
+    """A class to record timesketch analyzer output.
+
+    Attributes:
+        platform (str): [Required] Analyzer platfrom.
+        analyzer_identifier (str): [Required] Unique analyzer identifier.
+        analyzer_name (str): [Required] Analyzer display name.
+        result_status (str): [Required] Analyzer result status.
+            Valid values are success, no-findings, error.
+        dfiq_question_id (str): [Optional] Digital Forensics Investigative
+            Question (DFIQ) questions ID.
+        dfiq_question_conclusion (str): [Optional] DFQI questions answer/
+            conclusion.
```

Comment:
I opt to remove this for now. DFIQ is still in flux and we can add this later if needed.

An analyzer (analysis DB object) will be part of the DFIQ models, so we will have this link on that layer.

Reply:
Even when the analyzer is part of the DFIQ models, this would only work TS internally. The reference would get lost when the results are exported to any other tool.

I'm fine with removing them for now, but we need to revisit how we want them added in the normed analyzer output later on again, when they become a thing.

Reply:
Removed DFIQ attributes in analyzer output class.